### ISSUE TITLE
Import fails on MediaWiki 1.37 "content model 'sanitized-css' is not registered on this wiki"

### ISSUE THREAD

**Issue by sjlyoung58** [hallowelt/migrate-confluence#47]
TLDR - I encountered the issue below, but I have worked around it and completed the import - great job!

I ran up a new docker Mediawiki 1.37 instance and installed/enabled extensions as per instructions - 

Semantic MediaWiki
HeaderTabs
ParserFunctions

Images imported fine first time.

Upon importing output.xml I hit an issue with `The content model 'sanitized-css' is not registered on this wiki.`

```root@afba515e3237:/var/www/html# php maintenance/importDump.php result/output.xml
MWUnknownContentModelException from line 203 of /var/www/html/includes/content/ContentHandlerFactory.php: The content model 'sanitized-css' is not registered on this wiki.
See https://www.mediawiki.org/wiki/Content_handlers to find out which extensions handle this content model.
#0 /var/www/html/includes/content/ContentHandlerFactory.php(272): MediaWiki\Content\ContentHandlerFactory->validateContentHandler('sanitized-css', NULL)
#1 /var/www/html/includes/content/ContentHandlerFactory.php(191): MediaWiki\Content\ContentHandlerFactory->createContentHandlerFromHook('sanitized-css')
#2 /var/www/html/includes/content/ContentHandlerFactory.php(97): MediaWiki\Content\ContentHandlerFactory->createForModelID('sanitized-css')
#3 /var/www/html/includes/import/WikiImporter.php(1311): MediaWiki\Content\ContentHandlerFactory->getContentHandler('sanitized-css')
#4 /var/www/html/includes/import/WikiImporter.php(1063): WikiImporter->getContentHandler('sanitized-css')
#5 /var/www/html/includes/import/WikiImporter.php(1087): WikiImporter->makeContent(Object(Title), 0, Array)
#6 /var/www/html/includes/import/WikiImporter.php(988): WikiImporter->processRevision(Array, Array)
#7 /var/www/html/includes/import/WikiImporter.php(921): WikiImporter->handleRevision(Array)
#8 /var/www/html/includes/import/WikiImporter.php(738): WikiImporter->handlePage()
#9 /var/www/html/maintenance/importDump.php(354): WikiImporter->doImport()
#10 /var/www/html/maintenance/importDump.php(286): BackupReader->importFromHandle(Resource id #912)
#11 /var/www/html/maintenance/importDump.php(130): BackupReader->importFromFile('result/output.x...')
#12 /var/www/html/maintenance/doMaintenance.php(108): BackupReader->execute()
#13 /var/www/html/maintenance/importDump.php(359): require_once('/var/www/html/m...')
#14 {main}
```

I looked at output.xml and found 2 references, these weren't in my content pages, so I xml commented these out

```
  <!--page>
    <title>Template:Layout.css</title>
    <revision>
      <model>sanitized-css</model>
      <format>text/css</format>
      <text>.ac-layout-section {
	display: flex;
}
---snip---
  </page-->
---snip---
  <!--page>
    <title>Template:SimplePanel.css</title>
    <revision>
      <model>sanitized-css</model>
      <format>text/css</format>
      <text>.simplepanel {
	border: 2px solid;
	border-radius: 3px;
	padding: 0.5em;
}
---snip---
  </page-->
```

The import then ran through fine and everything is great on Mediawiki!

```
root@afba515e3237:/var/www/html# php maintenance/importDump.php result/output.xml
100 (4.25 pages/sec 4.25 revs/sec)
Done!
You might want to run rebuildrecentchanges.php to regenerate RecentChanges,
and initSiteStats.php to update page and revision counts
root@afba515e3237:/var/www/html# php maintenance/rebuildrecentchanges.php
Rebuilding $wgRCMaxAge=7776000 seconds (90 days)
Clearing recentchanges table for time range...
Loading from page and revision tables...
Inserting from page and revision tables...
Updating links and size differences...
Loading from user and logging tables...
Flagging bot account edits...
Flagging auto-patrolled edits...
Removing duplicate revision and logging entries...
Deleting feed timestamps.
Done.
root@afba515e3237:/var/www/html# php maintenance/initSiteStats.php --update
Refresh Site Statistics

Counting total edits...224
Counting number of articles...28
Counting total pages...188
Counting number of users...3
Counting number of images...70

Updating site statistics...done.

Done.
```

Some tidying up to do as expected, but this has saved me so much effort in resurrecting my old site from 2012 (Confluence 4) I can't thank you enough - great job!

**Comment by osnard** [comment]
Thanks for reporting. You are right. The "sanitized-css" content model is not part of MediaWiki Core. Please install "Extension:TemplateStyles" [1]. I have updated the `README.md` file accordingly [2]

[1] https://www.mediawiki.org/wiki/Extension:TemplateStyles
[2] https://github.com/hallowelt/migrate-confluence/commit/cb3fbd2a4ee26ab0f27091c00a3ec4dddfba1ee0
